massa-execution-worker/src/context.rs

```diff
@@ -146,16 +150,16 @@ impl ExecutionContext {
     /// # arguments
     /// * slot: slot at which the execution will happen
     /// * req: parameters of the read only execution
-    /// * previous_changes: list of ledger changes that happened since the final ledger state and before this execution
-    /// * final_ledger: thread-safe access to the final ledger. Note that this will be used only for reading, never for writing
+    /// * previous_changes: list of state changes that happened since the final state state and before this execution
```

Comment:
```suggestion
    /// * previous_changes: list of state changes that happened since the final_state state and before this execution
```

For better comprehension